### PR TITLE
Fe user profile final

### PR DESCRIPTION
**Pull request by Menubrea**
Final changes for this ``SPRINT``  to ``Profile`` for ``User related responsibilities``.

## Changelog
- Rewrote code for render functionality to be more ``DRY`` and future friendly. 
- Added edit as static HTML elements.
- Provided commentary for future developers.

### CODE REVIEW

File: pages/user/index.html [hunk 1]
```diff
@@ -12,8 +27,6 @@
     <script src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
```
Comment (Christonn93) [inline pages/user/index.html:27]: You need to add the author and team name on here. Ref to the front end team discussion board :) 
Love that you have added comments for the next class
Great work @Menubrea

Comment (Menubrea) [reply pages/user/index.html:27]: Isn't line 22 - 24 what you're looking for, or am I misunderstanding?
```html
    <!-- Dev -->
    <meta name="author" content="@Menubrea" />
    <meta name="team" content="User" />
```

Comment (Christonn93) [reply pages/user/index.html:27]: Thats correct, It did not show up for me.. 
image
Came up when i clicked this 
image

Sorry, my mistake!

Comment (Menubrea) [reply pages/user/index.html:27]: All good. I appreciate you checking over. If there's anything I can contribute with, let me know. I feel quite invested in this project.

**Pull request merged**